File: csharp/bypstest-ser/src-ser/byps/test/api/remote/RemoteConstants.cs
using System;
using System.Collections.Generic;
using System.Threading.Tasks;
using byps;

namespace byps.test.api.remote
{
	
	/// <summary>
	/// Interface functions to verify constant definitions.
	/// </summary>
	// checkpoint byps.gen.cs.GenRemoteClass:93
	public interface RemoteConstants : BRemote {
	
		void Compare_HebrewC(byps.test.api.cons.HebrewZ ALEPH, byps.test.api.cons.HebrewZ BETH);
		void Compare_HebrewC(byps.test.api.cons.HebrewZ ALEPH, byps.test.api.cons.HebrewZ BETH, BAsyncResult<Object> asyncResult) ;
		Task Compare_HebrewCAsync(byps.test.api.cons.HebrewZ ALEPH, byps.test.api.cons.HebrewZ BETH);
		
		bool Compare_AllTypesC(bool bool1s, bool bool2s, char char1s, String stringNull, char char2s, short short1s, int int1s, long long1s, float float1s, double double1s, String string1s, byps.test.api.cons.AllTypesZ ALL, int[] arrInt, int[,,,] arrInt4, String[] arrStrings, String[,,,] arrStrings4, byps.test.api.cons.AllTypesZ[,] arrAll);
		void Compare_AllTypesC(bool bool1s, bool bool2s, char char1s, String stringNull, char char2s, short short1s, int int1s, long long1s, float float1s, double double1s, String string1s, byps.test.api.cons.AllTypesZ ALL, int[] arrInt, int[,,,] arrInt4, String[] arrStrings, String[,,,] arrStrings4, byps.test.api.cons.AllTypesZ[,] arrAll, BAsyncResult<bool> asyncResult) ;
		Task<bool> Compare_AllTypesCAsync(bool bool1s, bool bool2s, char char1s, String stringNull, char char2s, short short1s, int int1s, long long1s, float float1s, double double1s, String string1s, byps.test.api.cons.AllTypesZ ALL, int[] arrInt, int[,,,] arrInt4, String[] arrStrings, String[,,,] arrStrings4, byps.test.api.cons.AllTypesZ[,] arrAll);
		
		
	}
}  // end namespace

File: java/bypstest-api/src/byps/test/api/remote/RemoteConstants.java
package byps.test.api.remote;

import byps.BRemote;
import byps.RemoteException;
import byps.test.api.cons.AllTypesZ;
import byps.test.api.cons.HebrewZ;

/**
 * Interface functions to verify constant definitions.
 *
 */
public interface RemoteConstants extends BRemote {

	void compare_HebrewC(HebrewZ ALEPH, HebrewZ BETH) throws RemoteException;
	
	boolean compare_AllTypesC(
			boolean bool1s, 
			boolean bool2s,
			char char1s,
			String stringNull,
			char char2s,
			short short1s,
			int int1s,
			long long1s,
			float float1s,
			double double1s,
			String string1s,
			AllTypesZ ALL,
			int[] arrInt,
			int[][][][] arrInt4,
			String[] arrStrings,
			String[][][][] arrStrings4,
			AllTypesZ[][] arrAll) throws RemoteException;
}
